### PR TITLE
Add standalone application that allows to load pbtxt graphs.

### PR DESCRIPTION
**Pull request by r4nt**

### CODE REVIEW

File: tensorboard/plugins/graph/standalone/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        width: 250px;
+        position: absolute;
+      }
+
+      paper-dialog ul {
+        padding: 0 0 0 50px;
+      }
+    </style>
+    <div id="side">
+      <tf-graph-controls devices-for-stats="{{_devicesForStats}}" color-by-params="[[_colorByParams]]" stats="[[_stats]]" color-by="{{_colorBy}}"
```
Comment (stephanwlee) [inline tensorboard/plugins/graph/standalone/index.html]: Where do these properties like `_devicesForStats` come from?

File: tensorboard/plugins/graph/standalone/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        render-hierarchy="{{_renderHierarchy}}" stats="[[_stats]]"></tf-graph-board>
+    </div>
+  </template>
+  <script>
+    Polymer({
+      is: "tf-graph-standalone",
+    });
+  </script>
+</dom-module>
+<tf-graph-standalone></tf-graph-standalone>>
```
Comment (stephanwlee) [inline tensorboard/plugins/graph/standalone/index.html]: omit the extra `>` at the end please

File: tensorboard/plugins/graph/standalone/index.html (hunk 1 above)
Comment (r4nt) [reply tensorboard/plugins/graph/standalone/index.html]: Those are as far as I understand how polymer connects them between different modules. For example, selectedFile is set in tf-graph-controls.html, but used in tf-graph-loader.html; if it is not connected here, loading a file will not work.

File: tensorboard/plugins/graph/standalone/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<script src="../../webcomponentsjs/webcomponents-lite.min.js"></script>
+<link rel="import" href="../tf-graph-controls/tf-graph-controls.html">
+<link rel="import" href="../tf-graph-loader/tf-graph-loader.html">
+<link rel="import" href="../tf-graph-board/tf-graph-board.html">
+<title>TF Graph Standalone</title>
+<dom-module id="tf-graph-standalone">
```
Comment (nfelt) [inline tensorboard/plugins/graph/standalone/index.html]: I believe the existing `<tf-graph-app>` component here is supposed to be usable for this purpose:
https://github.com/tensorflow/tensorboard/blob/1.10.0/tensorboard/plugins/graph/tf_graph_app/tf-graph-app.html
https://github.com/tensorflow/tensorboard/blob/1.10.0/tensorboard/plugins/graph/tf_graph_app/index.html

Would it be possible for you to use that existing one, potentially with fixes if you need them?

Comment (r4nt) [reply tensorboard/plugins/graph/standalone/index.html]: When I bazel run tensorboard/plugins/graph/tf_graph_app, and
a) open tf-graph-app.html: I get an empty page
b) open index.html: I get a page describing how to embed the app with a given pbtxt
From that experience I assumed that was not actually meant to be run just so a user could press a button to load a pbtxt :)

Comment (r4nt) [reply tensorboard/plugins/graph/standalone/index.html]: I can run the /demo there, and that gives me the demo graph, but there the 'Choose File' button is not connected. I could fix that if that makes more sense...

Comment (r4nt) [reply tensorboard/plugins/graph/standalone/index.html]: Fixed the demo app in the other patch, abandoning this one.

**Pull request closed without merging**